src/__tests__/useA11yBlockLink.test.js

```diff
@@ -0,0 +1,144 @@
+import React, {useRef} from 'react';
+import {render, configure, fireEvent} from '@testing-library/react';
+import useA11yBlockLink from '../useA11yBlockLink';
+
+configure({testIdAttribute: 'data-test-id'});
+
+const Demo = ({testId, handleMainClick, handleThirdClick}) => {
+  const mainClickRef = useRef();
+  const secondRef = useRef();
+  const thirdRef = useRef();
+
+  const {handleClick: handleA11yClick} = useA11yBlockLink({
```

Comment:
Seems like `handleA11yClick` isn't called anywhere either 🤔

Reply:
Updated to make sure we do.